.github/workflows/codespell.yaml

```diff
@@ -0,0 +1,21 @@
+name: codespell
+on:
+  push:
+jobs:
+  codespell:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Codespell with annotations
+        uses: codespell-project/actions-codespell@v1
+        with:
+          builtin: clear,rare,informal
```

Comment:
Will this not use the `.codespellrc` that is included in the repository?

Reply:
Inconveniently, no. Although, I do believe the maintainer has a [PR](https://github.com/codespell-project/actions-codespell/pull/19) that should support this in a future version of the action.